src/refda/amm/oper.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2011-2024 The Johns Hopkins University Applied Physics
+ * Laboratory LLC.
+ *
+ * This file is part of the Delay-Tolerant Networking Management
+ * Architecture (DTNMA) Tools package.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef REFDA_AMM_OPER_H_
+#define REFDA_AMM_OPER_H_
+
+#include "refda/eval.h"
+#include "refda/eval_ctx.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// forward declaration for callback reference
+struct refda_amm_oper_desc_s;
+typedef struct refda_amm_oper_desc_s refda_amm_oper_desc_t;
+
+/** A control (CTRL) descriptor.
```

Comment:
**Question**: Why do we call this `oper` if it is a `CTRL` descriptor?

Reply:
this was a typo